index.html

```diff
@@ -95,78 +95,62 @@ <h3 class="ability_header">Skills</h3>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
+                    <ul class="languages projectOne">
+                        <li></li>
+                        <li></li>
+                        <li></li>
+                        <li></li>
                     </ul>
-                    <button type="button" class="btn">See Project</button>
+                    <button type="button" class="btn openPopUp"></button>
                 </div>
             </div>
 
             <div class="work_element work_element_reverse">
-                <img class="card" src="images/placeholder.png" alt="placeholder for pending images">
                 <div class="supporting_text">
-                    <h3>Multi-Post Stories</h3>
-                    <p class="work_info">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
+                    <ul class="languages projectTwo">
+                        <li></li>
+                        <li></li>
+                        <li></li>
+                        <li></li>
                     </ul>
-                    <button type="button" class="btn">See Project</button>
+                    <button type="button" class="btn openPopUp"></button>
                 </div>
             </div>
 
             <div class="work_element">
-                <img class="card" src="images/placeholder.png" alt="placeholder for pending images">
                 <div class="supporting_text">
-                    <h3>Multi-Post Stories</h3>
-                    <p class="work_info">
-                        A daily selection of privately personalized reads; no accounts or sign-ups required.
-                        has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-                    </p>
-                    <ul class="languages">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>Bootstrap</li>
-                        <li>Ruby</li>
+                    <h3></h3>
+                    <p class="work_info"></p>
+                    <ul class="languages projectThree">
+                        <li></li>
+                        <li></li>
+                        <li></li>
```

Comment:
- [ ] please kindly make the mobile version full height like that of figma.
yours
![Screenshot (439)_LI](https://user-images.githubusercontent.com/65298048/188986641-8d3deb18-5f64-413b-886c-6daa2a57926a.jpg)
figma
![Screenshot (438)](https://user-images.githubusercontent.com/65298048/188986788-8d4bf26a-d182-49af-995b-2bb36d205c7a.png)

Reply:
Implemented 👍